### PR TITLE
add "verify_ssl" param

### PR DESCRIPTION
**Pull request by vvzvlad**
The option verify_ssl=False has been added for test runs or runs within local env

### CODE REVIEW

File: grist_api/grist_api.py [hunk 1]
```diff
@@ -52,16 +52,26 @@ class GristDocAPI(object):
...
     the doc. The API key, if omitted, is taken from GRIST_API_KEY env var, or ~/.grist-api-key file.
+    
+    Parameters:
+    - doc_id (str): The document ID from Grist (the part of the URL after /doc/).
+    - api_key (str, optional): API key for authentication. If not provided, it will be read from
+      the environment variable GRIST_API_KEY or from the file ~/.grist-api-key.
+    - server (str, optional): Base URL of the Grist server. Defaults to 'https://api.getgrist.com'.
+    - dryrun (bool, optional): If True, no actual changes will be made to the document. Defaults to False.
+    - verify_ssl (bool, optional): If True (default), SSL certificates will be verified. Set to False
+      to disable SSL verification (use with caution).
```
Comment (dsagal) [inline grist_api/grist_api.py:68]: Maybe add a hint about purpose, like `(use with caution, e.g. for local development or testing)`.

File: grist_api/grist_api.py [hunk 2]
```diff
@@ -52,16 +52,26 @@ class GristDocAPI(object):
...
     """
     self._dryrun = dryrun
     self._server = server
     self._api_key = api_key or get_api_key()
     self._doc_id = doc_id
+    self.verify_ssl = verify_ssl
```
Comment (dsagal) [inline grist_api/grist_api.py:74]: I think it would be better to make it a private member, with a leading underscore, like for the member above.

**Review by dsagal — COMMENTED** [review]
Thank you!

**Comment by vvzvlad** [comment]
Done

**Review by dsagal — APPROVED** [review]

**Pull request merged**